rcl_action/include/rcl_action/action_server.h

```diff
@@ -161,7 +162,8 @@ rcl_action_get_zero_initialized_server(void);
  * \param[out] action_server handle to a preallocated, zero-initialized action server structure
  *   to be initialized.
  * \param[in] node valid node handle
- * \param[in] clock valid clock handle
+ * \param[in] expire_timer An initialized and canceled timer. The caller must guarantee the lifetime of the
```

Comment:
this is weird, uncrustify should warn this because maximum line length is 100 characters...